packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -0,0 +1,364 @@
+<template>
+    <div
+        ref="cookieBanner"
+        :class="[$style['c-cookieBanner'], $style['c-cookieBanner-overlay'], { [$style['c-cookieBanner--hidden']]: hideBanner }]"
+        data-cookie-consent-overlay
+        data-test-id="cookieBanner-component"
+        :aria-hidden="hideBanner">
+        <div
+            v-if="!legacyBanner"
+            :class="[$style['c-cookieBanner-card'], { [$style['c-cookieBanner-ios']]: checkIosSafari() }]">
+            <div
+                :class="$style['c-cookieBanner-content']"
+                role="dialog"
+                aria-labelledby="cookieConsentLabel"
+                aria-modal="true">
+                <h2
+                    id="cookieConsentLabel"
+                    :class="$style['c-cookieBanner-title']">
+                    <a
+                        ref="cookieBannerHeadingFocus"
+                        name="cookieConsentTitle"
+                        tabindex="0"
+                        data-consent-title>
+                        {{ $t('mainTitle') }}
+                    </a>
+                </h2>
+
+                <p :class="$style['c-cookieBanner-text']">
+                    {{ $t('textLine1') }}
```

Comment:
Do we have some more descriptive names for these? `textLine1`, `textLine2` and ``textLine3` don't say a lot.

Reply:
@SachaZvetelman @ashleynolan I have been racking my brain all night on this one. I could rename the vars `cookieBannerTextLine1` etc, but that seems overly complex given that we know this is the cookie banner. I have tried to get some sense from the translations of what is in each line, but each language differs slightly (these are translations not transliterations, so word order is not the same). Maybe `topLineText`, `middleLineText` and 'bottomText`? But these seem to be just a variation on 1,2 and 3, and we might at some point decide on different screen positions. 

We can maybe discuss it in the a11y meeting I'm setting up (to discuss the focus issue).

Reply:
@simonsandersje, it was a minor comment, I didn't want you to rack your brain all night on this thing! If we cannot find an easy, straight-forward name, leave it as it is. It's not a big deal at all. I'm marking this conversation as resolved. 🙂